tests/mocks.py

```diff
@@ -0,0 +1,22 @@
+from collections import namedtuple
+
+from azure.core.credentials import TokenCredential
+
+MockToken = namedtuple("MockToken", ["token", "expires_on"])
+
+
+class MockAzureCredential(TokenCredential):
+    def get_token(self, uri):
+        return MockToken("", 9999999999)
+
```

Comment:
did you notice this? see the test below
it errored as it's not supposed to be an AsyncTokenCredential